balancer-js/src/modules/pools/apr/apr.integration.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('APR tests', () => {
       const pool = await pools.find(veBalId);
       if (pool) {
         const apr = await pools.apr(pool);
-        expect(apr.protocolApr).to.be.greaterThan(1);
+        expect(apr.protocolApr).to.be.greaterThanOrEqual(1);
```

Comment:
Are you sure this change is ok? I have a feeling that rates should always be > 1, or they don't make much sense in the first place.

Reply:
I'm not sure, but I asked Jarek about this change, as soon as he answer I'll confirm

Reply:
Removed this